integreat_cms/static/src/js/media-management/component/breadcrumbs.tsx

```diff
@@ -18,6 +18,10 @@ type Props = {
 
 const Breadcrumbs = ({ breadCrumbs, searchQuery, mediaTranslations, allowDrop, dropItem }: Props) => {
     const [currentDragTarget, setCurrentDragTarget] = useState<null | number>(null);
+
+    const actualUrl = window.location.href.split("/");
+    const isUnusedFiltered = actualUrl[actualUrl.length - 2] === "filter-unused-media-files";
```

Comment:
I'd rather add a new property here that's passed from `UnusedFilterResultLibrary`.